content/account/dashboard/api.textile

```diff
@@ -0,0 +1,52 @@
+---
+title: API keys
+meta_description: “Manage Ably API keys by creating, updating, setting restrictions, and exploring integration options.”
+meta_keywords: "API keys, Ably, create key, update key, key restrictions, integration, webhooks, authentication"
+---
+
+The API keys tab lists all API keys associated with your account and provides information on each key's capabilities and restrictions. You can "create a new API key":#create and manage an existing one.
+
+h2(#create). Create a new API key
+
+Click  *create new API key*. During the _create_ process, you can:
+* Assign a friendly name.
+* Give the new API key a descriptive name (e.g. chat app key) so it is easy to identify later.
+
+h2(#manage). Manage an API key
+
+To manage an API key, you set capabilities, define resource restrictions, use revocable tokens for security, and adjust key settings as needed.
```

Comment:
Does this need a new section? Is it not just part of creating an API key? Having it as its own section makes it look like a separate operation that you do after creating one.